curated_transformers/tokenization/bbpe_tokenizer.py

```diff
@@ -0,0 +1,131 @@
+from typing import List, Type, TypeVar
+from cutlery import ByteBPEProcessor
+import json
+from pathlib import Path
+
+from . import PiecesWithIds, Tokenizer
```

Comment:
```suggestion
from .tokenizer import PiecesWithIds, Tokenizer
```
Potential circular import?